dislib/trees/forest.py

```diff
@@ -8,13 +8,14 @@
 from sklearn.base import BaseEstimator
```

Comment:
please consider exposing only objects that are necessary in dislib/regression/__init__.py 
(currently all of them are exposed)